StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/ReadabilityRules/SA1130CodeFixProvider.cs

```diff
@@ -38,19 +38,34 @@ public override FixAllProvider GetFixAllProvider()
         }
 
         /// <inheritdoc/>
-        public override Task RegisterCodeFixesAsync(CodeFixContext context)
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             foreach (var diagnostic in context.Diagnostics)
             {
+                if (await CanFixAsync(context, diagnostic).ConfigureAwait(false))
```

Comment:
❗️ This is either a bad method name or the expression is inverted

```suggestion
                if (!(await CanFixAsync(context, diagnostic).ConfigureAwait(false)))
```